controlplane/rosa/api/v1beta2/rosacontrolplane_types.go

```diff
@@ -180,6 +180,67 @@ type RosaControlPlaneSpec struct { //nolint: maligned
 	// ControlPlaneEndpoint represents the endpoint used to communicate with the control plane.
 	// +optional
 	ControlPlaneEndpoint clusterv1.APIEndpoint `json:"controlPlaneEndpoint"`
+
+	// ClusterRegistryConfig represents registry config used with the cluster.
+	// +optional
+	ClusterRegistryConfig *RegistryConfig `json:"clusterRegistryConfig,omitempty"`
+}
+
+// RegistryConfig for ROSA-HCP cluster
+type RegistryConfig struct {
+	// AdditionalTrustedCa containing the registry hostname as the key, and the PEM-encoded certificate as the value,
+	// for each additional registry CA to trust.
+	// +optional
+	AdditionalTrustedCa map[string]string `json:"additionalTrustedCa,omitempty"`
```

Comment:
Lets name this field `AdditionalTrustedCAs`. That will let the user know they can add more than one, and match the semantics of other abbreviations on fields.

Reply:
okay, done.